library/tasks/HISAT2.wdl

```diff
@@ -240,8 +241,11 @@ task HISAT2SingleEnd {
       --new-summary --summary-file "${output_basename}.log" \
       --met-file ${output_basename}.hisat2.met.txt --met 5 \
       --seed 12345 \
+      -k 10 \
+      --secondary \
```

Comment:
Why add secondary only in single end? I would argue for keeping this consistent with pe.

Reply:
Paired end does use secondary. These parameters were added to the hisat call matches the pe call. I think this task was unused and had diverged from the pe task.